packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -211,6 +212,23 @@ type HydrationHostConfig<T, P, I, TI, C, CX, PL> = {
   ): void,
 };
 
+// 0 is PROD, 1 is DEV.
+// Might add PROFILE later.
+type BundleType = 0 | 1;
+
+type DevToolsConfig<I, TI> = {|
+  bundleType: BundleType,
+  version: string,
+  rendererPackageName: string,
+  // Note: this actually *does* depend on Fiber internal fields.
+  // Used by "inspect clicked DOM element" in React DevTools.
+  findFiberByHostInstance?: (instance: I | TI) => Fiber,
+  // Used by RN in-app inspector.
+  // This API is unfortunately RN-specific.
+  // TODO: Change it to accept Fiber instead and type it properly.
+  getInspectorDataForViewTag?: (tag: number) => Object,
```

Comment:
Can you remind me where it is used?

Reply:
It used to be part of the devtools config I think. https://github.com/facebook/react-devtools/blob/ef493abba0d08c0243bdb1f2183edc797d5d366c/packages/react-devtools-core/src/backend.js#L13-L18

I _think_ the usage was so that an app could indicate whether it should take over the devtools connection or not, but not really sure. Or maybe to return false while still starting up (according to this comment)

https://github.com/facebook/react-devtools/blob/ef493abba0d08c0243bdb1f2183edc797d5d366c/packages/react-devtools-core/src/backend.js#L52-L55

Reply:
It's not required. I added it for RN so that multiple apps on simulator don't steal devtools from each other.

Reply:
Also, it's part of the *options* to `connectToDevTools()` call which sets up a websocket connection.

This is not related to the `injectIntoDevTools` call which injects renderer into a global variable.